### PR TITLE
[Merged by Bors] - Store and use client TLS for connectors

### PR DESCRIPTION
**Pull request by nacardin**
Resolves https://github.com/infinyon/fluvio/issues/1970

### CODE REVIEW

**Comment by nacardin** [comment]
The `fs_group` is needed for `fluvio` user to read mounted secret (client TLS certs).

File: crates/fluvio/src/config/tls.rs [hunk 1]
```diff
@@ -122,18 +123,38 @@ pub struct TlsCerts {
...
+}
+
 impl TlsCerts {
     /// Attempts to write the inline TLS certs into temporary files
     ///
     /// Returns a `TlsPaths` populated with the paths where the
     /// temporary files were written.
     pub fn try_into_temp_files(&self) -> Result<TlsPaths, IoError> {
         use std::fs::write;
-        let tmp = std::env::temp_dir();
```
Comment (sehz) [inline crates/fluvio/src/config/tls.rs:132]: Is there a reason for writing a cert in the client rather than installer?

**Review by sehz — CHANGES_REQUESTED** [review]
Looks good. Just have one issue

**Comment by sehz** [comment]
CI failures. I think it is better to move certs generation to the installer

File: crates/fluvio/src/config/tls.rs (hunk 1 above)
Comment (nacardin) [reply crates/fluvio/src/config/tls.rs:132]: It was probably done to keep it in the impl of TlsCerts, but it makes more sense to have this is installer. Will fix

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**